packages/react/src/components/Button/index.js

```diff
@@ -6,13 +6,21 @@ import { INTENTS } from "../../constants";
 
 const intents = Object.values(INTENTS);
 
-const Button = ({ as: Component, className, disabled, intent, ...props }) => (
+const Button = ({
+  as: Component,
+  className,
+  disabled,
+  forwardedRef,
+  intent,
+  ...props
+}) => (
   <Component
     className={cx("Button", className, {
       "is-disabled": disabled,
       [`is-${intent}`]: intent,
     })}
     disabled={disabled}
+    ref={forwardedRef}
```

Comment:
Ohh, so we still will be needing to remember about that when implementing new components, huh?

Reply:
Is it really vital to do so since you're doing 
```
        React.cloneElement(children, {
          className: "Popover-wrapper",
          forwardedRef: ref,
```
?

Not saying it's not. Just trying to understand

Reply:
Yeah we still need to do that because otherwise `forwardedRef` will end on the HTML element (via `{...props}`) and React will throw an error because this prop is not recognized as valid prop